app/views/works/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<main>
+<h1>List of Works</h1>
+<section class="media-table">
+<%['movie', 'book', 'album'].each do |category| %>
+<h4><%=category.capitalize%>s </h4>
+<table class="table">
+  <thead>
+  <tr class="header_font">
+    <th>Votes</th>
+    <th>Title</th>
+    <th>Created By</th>
+    <th>Published</th>
+    <th>Upvote</th>
+  </tr>
+</thead>
+<tbody>
+  <% @works.each do |work| %>
```

Comment:
Notice that you do a very similar table here a lot.  You could DRY this out with a partial.